qadaptive_avgpool2d benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29274 qadaptive_avgpool2d benchmarking**
* #29268 qavgpool benchmarking
* #29257 Adding short tests to qmaxpool benchmarking

Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
qadaptive_avgpool2d benchmarking

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qpool_test.py:
```diff
@@ -89,8 +119,20 @@ def init(self, N, C, H, W, k, s, p, ceil, contig, dtype):
 op_bench.generate_pt_test(qmaxpool2d_short_configs, QAvgPool2dBenchmark)
 op_bench.generate_pt_test(qmaxpool2d_short_configs, QMaxPool2dBenchmark)
+
+op_bench.generate_pt_test(qadaptive_avgpool2d_long_configs,
```
Concatenate configs.

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Reduced the number of tests on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qpool_test.py:
```diff
@@ -75,20 +97,29 @@ def forward(self):
-        self.pool_op = torch.nn.AvgPool2d(kernel_size=k, stride=s, padding=p,
-                                          ceil_mode=ceil)
+    def init(self, N, C, H, W, k, s, p, contig, dtype):
+        self.pool_op = torch.nn.MaxPool2d(kernel_size=k, stride=s, padding=p,
```
Should this be AvgPool2d? Seems to be a mismatch?

z-a-f replied:
You are right -- I noticed it yesterday, forgot to stack tthe changes :)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Rebasing on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

Commit:
Update on "qadaptive_avgpool2d benchmarking"


Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

mingzhe09088 reviewed (approved):
LGTM!
